packages/scripts/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 - Added support for `build` script ([#12837](https://github.com/WordPress/gutenberg/pull/12837))
 - Added support for `start` script ([#12837](https://github.com/WordPress/gutenberg/pull/12837))
+- The bundled `npm-package-json-lint` dependency has been updated from requiring `^3.3.1` to requiring `^3.6.0`, adding [a rule which enforces the presence of the `repository.directory` field](https://github.com/tclindner/npm-package-json-lint/pull/106) [#14200](https://github.com/WordPress/gutenberg/pull/14200)
```

Comment:
```suggestion
- Updated `npm-package-json-lint` dependency.
```

Per my comment above this isn't explicitly required, only the _root_ `package.json` file is required to be updated.

The _scripts_ package changelog only requires a note that the `npm-package-json-lint` dependancy was updated.